lib/webauthn/attestation_statement/fido_u2f.rb

```diff
@@ -51,6 +69,14 @@ def valid_signature?(authenticator_data, client_data_hash)
           .verify(signature, verification_data(authenticator_data, client_data_hash))
       end
 
+      def certificate_chain_trusted?
+        find_metadata
+        return false unless metadata_statement
```

Comment:
Unfortunately this fails conformance testing at the moment because `metadata.zip` doesn't contain entries/statements with the `attestationCertificateKeyIdentifiers` array containing any strings: https://github.com/fido-alliance/conformance-tools-issues/issues/501#issuecomment-523252429